index.html

```diff
@@ -9,43 +9,58 @@
     <link rel="stylesheet" href="style/style.css">
     <title>Portfolio</title>
     <style>
-        section{
-            background: url("header-shapes mobile.png");
+        .info{
+            background: url("style/img/header-shapes\ mobile.png");
             background-size: cover;
+            display: flex;
+            flex-direction: column;
+            align-items: flex-start;
+            padding: 114px 24px;
+            gap: 12px;
+            position: absolute;
+            width: 375px;
+            height: 604px;
+            left: calc(50% - 375px/2);
+            top: 92px;
         }
     </style>
 </head>
 <body>
     <header>
         <nav>
-            <input type="checkbox" id="check">
-            <label for="check" class="checkbtn">
-                <i class="fas fa-bars"></i>
-            </label>
-            <label class="logo">My logo</label>
-            <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
-            </ul>
+            <div class="logo">
+                <h3 class="myLogo">
+                    <a href="#Home">My Logo</a>
+                </h3>
+            </div>
+            <div class="hamburger">
+                <a href="#"><img src="style/img/hamburger-vactors.png" class="hamburger-img"></a>
+            </div>
         </nav>
     </header>
 
-    <section>
-        <h1 class="title">I'm Salwa <br>Glad to see you!</h1>
-        <p class="presentation">I’m a software developer! I can help you build a product , feature or
-            website Look through some of my work and experience! If you like what
```

Comment:
- [Optional] Consider using an indentation of two spaces consistently in all your documents. This is a conventional and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).